modules/programs/msmtp/default.nix

```diff
@@ -66,9 +55,36 @@ in
 
       package = lib.mkPackageOption pkgs "msmtp" { };
 
+      configContent = mkOption {
+        default = "";
+        type = types.lines;
+        example = lib.literalExpression ''
+          lib.mkOrder 1200 ''''
+            set syslog
+          '''';
+        '';
+        description = ''
+          Content added to msmtp config.
+          See <https://marlam.de/msmtp/msmtprc.txt> for examples.
+
+          Note, if running msmtp fails with the error message "account default
+          was already defined" then you probably have an account command here.
+          Account commands should be placed in
+          [](#opt-accounts.email.accounts._name_.msmtp.extraConfig).
+        '';
+      };
+
       extraConfig = mkOption {
         type = types.lines;
         default = "";
+        visible = false;
+        apply =
+          x:
+          lib.warnIfNot (x == "") ''
+            `programs.msmtp.extraAccounts` is deprecated, use `programs.msmtp.configContent` instead.
```

Comment:
```suggestion
            `programs.msmtp.extraConfig` is deprecated, use `programs.msmtp.configContent` instead.
```

Reply:
you are like a hawk, nothing escapes you ;) Thanks a lot for the good reviews !